### ISSUE TITLE
Problems with train

### ISSUE THREAD

**Issue by MC-E** [puruBHU/YOLSE_keras#1]
Hi,I am doing a project of detecting fingertip,can you tell me how to train this model and the dataset that needed.Thank you,looking forward to your reply.

**Comment by puruBHU** [comment]
I have trained the model with SCUT-Ego-Gesture dataset. The dataset is available at http://www.hcii-lab.net/data/SCUTEgoGesture/index.htm. The data preprocessing code is there in dataloader folder. I have trained the model in Keras. Please follow the code. And If you have any problem in understanding the code let me know.

**Comment by MC-E** [comment]
Thank you very much,I will try it immediately.

**Comment by MC-E** [comment]
Hi,I download the dataset(including:EgoGesture_JPG and label) but I do not konw how to deal with this dataset.

**Comment by MC-E** [comment]
The labels are all ".txt" file,how to convert them to ".csv" file?

**Comment by puruBHU** [comment]
Use pandas to read .txt file
